declarations.d.ts

```diff
@@ -63,6 +63,29 @@ interface IiTunesConnectApplication {
 }
 
 /**
+ * Describes configuration settings that modify the behavior of some methods.
+ */
+interface IConfigurationSettings {
+	/**
+	 * This string will be used when constructing the UserAgent http header.
+	 * @type {string}
+	 */
+	userAgentName: string;
```

Comment:
it's used for constructing `User-Agent` but the usage suppose you'll pass `name` of the client, shouldn't we change it to `clientName`?
For example in the future we may use this setting for other messages